src/fides/api/schemas/messaging/messaging.py

```diff
@@ -416,41 +416,64 @@ class MessagingConfigStatusMessage(BaseModel):
     detail: Optional[str] = None
 
 
-class MessagingTemplateBase(BaseModel):
+class BasicMessagingTemplateBase(BaseModel):
     type: str
     content: Dict[str, Any]
 
 
-class MessagingTemplateRequest(MessagingTemplateBase):
+class BasicMessagingTemplateRequest(BasicMessagingTemplateBase):
     pass
 
 
-class MessagingTemplateResponse(MessagingTemplateBase):
+class BasicMessagingTemplateResponse(BasicMessagingTemplateBase):
     label: str
 
 
-class BulkPutMessagingTemplateResponse(BulkResponse):
-    succeeded: List[MessagingTemplateResponse]
+class BulkPutBasicMessagingTemplateResponse(BulkResponse):
+    succeeded: List[BasicMessagingTemplateResponse]
     failed: List[BulkUpdateFailed]
 
 
 class MessagingTemplateWithPropertiesBase(BaseModel):
-    id: Optional[str]  # Since summary returns db or defaults, this can be null
+    id: str
     type: str
     is_enabled: bool
     properties: Optional[List[MinimalProperty]]
 
+    class Config:
+        orm_mode = True
+        use_enum_values = True
+
+
+class MessagingTemplateDefault(BaseModel):
+    type: str
+    is_enabled: bool
+    content: Dict[str, Any]
+
 
 class MessagingTemplateWithPropertiesSummary(MessagingTemplateWithPropertiesBase):
-    pass
+    class Config:
+        orm_mode = True
+        use_enum_values = True
 
 
 class MessagingTemplateWithPropertiesDetail(MessagingTemplateWithPropertiesBase):
     content: Dict[str, Any]
 
+    class Config:
+        orm_mode = True
+        use_enum_values = True
+
 
 class MessagingTemplateWithPropertiesBodyParams(BaseModel):
 
     content: Dict[str, Any]
```

Comment:
I just learned this during my last feature! Pydantic's `Field` class let's your specify an `example` that is picked up nicely by Swagger
```suggestion
    content: Dict[str, Any] = Field(
        example={
            "subject": "Message subject",
            "body": "Custom message body",
        }
    )
```
![image](https://github.com/ethyca/fides/assets/2092272/6c9a8da5-2812-4946-babc-164a8f9c8b96)

Or we could create a new schema for the content since the keys will always be `subject` and `body`